tracking/translations_parser/cli/experiments.py

```diff
@@ -71,16 +75,73 @@ def parse_experiment(
 
 
 def publish_group_logs(
```

Comment:
I don't want to block the merging since it adds functionality we need but next we touch this function it should be quickly refactored by extracting multiple functions.